pkg/components/exporter/exporter.go

```diff
@@ -489,6 +489,14 @@ func NewServiceMonitor() *monv1.ServiceMonitor {
 		TargetLabel: "instance",
 	}}
 
+	metricRelabelings := []*monv1.RelabelConfig{{
+		Action: "replace",
+		SourceLabels: []monv1.LabelName{
+			"container_namespace",
+		},
+		TargetLabel: "namespace",
+	}}
+
```

Comment:
@BGrasnick Have you tested this on OpenShift? IIUC, this will not be honoured in openshift if you use User Project Monitoring.